packages/fxa-settings/src/pages/Signin/SigninUnblock/index.test.tsx

```diff
@@ -151,55 +152,53 @@ describe('SigninUnblock', () => {
     it('with an empty code field', async () => {
       renderWithSuccess();
       const submitButton = screen.getByRole('button', { name: 'Continue' });
+      expect(submitButton).toBeDisabled();
+
+      // Button should be disabled, so clicking it should not trigger submission
```

Comment:
We have lots of tests that assert that submission happens so it seems like this has been around a while.